tasks/lzards-backup/src/index.ts

```diff
@@ -288,6 +315,9 @@ export const backupGranule = async (params: {
       granuleId: granule.granuleId,
     })));
   } catch (error) {
+    if (error.name === 'CollectionIdentifiersNotProvidedError') {
+      log.error(`${granule.granuleId}: Granule (${granule.granuleId}) will not be backed up.`);
+    }
     if (error.name === 'CollectionNotDefinedError') {
```

Comment:
Should this be `else if`? I don't think it really matters, just that we're checking the same property in both.

Reply:
Yes definitely https://github.com/nasa/cumulus/pull/3043/commits/ad477d13003920919959a9deef5c8793b2e4cd09#diff-31ab5953f1d25d2a7e338fb25acc5c7ae53aacc8cde330d0084eb4e00236cf3fR328